proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties

```
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś już połączony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
```

(minor) I believe this sounds better/more correct, roommate agrees :)
The same way you have below `Już próbujesz ...`
```suggestion
velocity.error.already-connected=Już jesteś połączony z tym serwerem!
velocity.error.already-connected-proxy=Już jesteś połączony z tym proxy!
```

---

I disagree on this. `Jesteś już` sounds more naturally to me.

---

At first, I felt "Jesteś już" was more natural but after some consideration, I think "Już jesteś" sounds a bit better.